### PR TITLE
(chore) Installation of the  ReactHookForm  and Zod dependencies

### PR DESCRIPTION
**Pull request by lumuchris256**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary
Installation of the initial necessary project dependencies of react hook form and Zod 

## Related Issue
https://issues.openmrs.org/browse/O3-2234

## Other
cc @hadijahkyampeire @denniskigen @vasharma05

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,5 +75,10 @@
   "resolutions": {
     "sass": "^1.54.3"
   },
-  "packageManager": "yarn@3.6.0"
+  "packageManager": "yarn@3.6.0",
+  "dependencies": {
+    "@hookform/resolvers": "^3.1.1",
+    "react-hook-form": "^7.45.1",
+    "zod": "^3.21.4"
+  }
```
Comment (vasharma05) [inline package.json:83]: Can these be added to `devDependencies`?

Comment (vasharma05) [reply package.json:83]: CC: @denniskigen

Comment (denniskigen) [reply package.json:83]: No, these shouldn't need to be devDeps.

**Review by denniskigen — APPROVED** [review]
LGTM.

**Pull request merged**